WALMA-15: Adding user, security and workflow related shortcuts

dministro opened the pull request:
WALMA-15

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Adding GoToContentItemDisplayByIdAsync

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/WALMA-15

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/WALMA-15

dministro committed:
Adding security shortcuts test
Updating submodule:
 - Lombiq.HelpfulLibraries: adding mvc attribute
 - Lombiq.UITestingToolbox: adding shortcuts

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/WALMA-15

dministro committed:
Adding test for workflow shortcut
Disabling dotnet-consolidation
Updating submodules:
 - Lombiq.HelpfulLibraries: null check
 - Lombiq.HelpfulExtensions: Nuget package version
 - Lombiq.UITestingToolbox: workflow related shortcut

Piedone commented on .github/workflows/build-and-test.yml:
```diff
@@ -14,6 +14,7 @@ jobs:
     with:
       machine-types: "[\"ubuntu-latest\", \"windows-latest\"]"
       timeout-minutes: 60
+      verify-dotnet-consolidation: false
```
I assume this is temporary and you'll remove it before this is ready for review?

dministro replied:
Yes, this is temporary

Piedone commented:
Use a PR title that makes sense in the context of OSOCE.

dministro committed:
Updating submodules:
 - Lombiq.HelpfulExtensions: Updating NuGet package versions
 - Lombiq.UITestingToolbox: Updating NuGet package versions

dministro committed:
Updating NuGet package versions
Restoring verify-dotnet-consolidation

dministro committed:
Restoring verify-dotnet-consolidation again

Luko6 commented:
Fix conflicts

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/WALMA-15

dministro committed:
Updating submodules:
 - Lombiq.HelpfulLibraries: Comment
 - Lombiq.UITestingToolbox: Renaming methods

dministro committed:
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating NuGet package versions
 - Lombiq.DataTables: Updating NuGet package versions
 - Lombiq.HelpfulExtensions: Updating NuGet package versions
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating NuGet package versions
 - Lombiq.Hosting.Tenants: Updating NuGet package versions
 - Lombiq.Privacy: Updating NuGet package versions
 - Lombiq.BaseTheme: Updating NuGet package versions

dministro committed:
Renaming AllowPermissionToRoleAsync to AddPermissionToRoleAsync
Renaming WorkflowsHttpEventGenerateUrlAsync to GenerateHttpEventUrlAsync
Updating submodule:
 - Lombiq.UITestingToolbox: Renaming methods

dministro committed:
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating NuGet package versions
 - Lombiq.DataTables: Updating NuGet package versions
 - Lombiq.HelpfulExtensions: Updating NuGet package versions
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating NuGet package versions
 - Lombiq.Hosting.Tenants: Updating NuGet package versions
 - Lombiq.Privacy: Updating NuGet package versions
 - Lombiq.BaseTheme: Updating NuGet package versions

Commit:
Updating submodules to "dev"
